src/boardModule.js

```diff
@@ -0,0 +1,177 @@
+import {
+  globals,
+} from './globals';
+import {
+  Player,
+} from './playerFactory';
+import {
+  displayModule,
+} from './displayModule';
```

Comment:
I think It's more readable having these lines in one line.
`import { globals } from '../globals';`

Reply:
solved